providers/implementations/ciphers/cipher_aes_gcm.c

```diff
@@ -20,7 +20,6 @@
 #include "prov/implementations.h"
 #include "prov/providercommon.h"
 
-#define AES_GCM_IV_MIN_SIZE     (64 / 8) /* size in bytes */
 /* Note: GCM_IV_MAX_SIZE is listed in ciphercommon_gcm.h */
```

Comment:
This comment here can be removed also - it is redundant..

Reply:
Fair enough.  The macro isn't mentioned in the file otherwise.